dynawo/3rdParty/CMakeLists.txt

```diff
@@ -521,11 +542,20 @@ else()
 
       CONFIGURE_COMMAND   <SOURCE_DIR>/bootstrap.sh
                           --without-icu
-                          --with-libraries=filesystem,program_options,serialization,system,log,iostreams
+                          --with-libraries=date_time,filesystem,program_options,serialization,system,log,iostreams,log_setup
+
+
 
       BUILD_COMMAND       <SOURCE_DIR>/b2
                           -j${NUMBER_OF_PROCESSORS}
-                          -sNO_BZIP2=1 cxxflags=-fPIC
+                          -sNO_BZIP2=1
+                          cxxflags=-fPIC
+                          --with-test
```

Comment:
why use this options ??? --with-libraries works well enough for me and it is redundant

Reply:
I told you I had to add it for compiling against boost 1.65. It works fine without it since we use 1.66. That makes I suspect test has not been done all the way with releases older than 1.66. It is hard to say "it works" in that case. Anyway I turned back to 1.66 and I will get rid of the extra --with-something.

Reply:
done

Reply:
once again I tested it in 1.65 and works fine for me so I really don't understand what is your error and you never showed it so it is difficult for me to grasp what is going wrong...